src/templates/base/2019/index.html

```diff
@@ -43,7 +43,100 @@ <h2>{{ self.intro_sub_title() }}</h2>
+    var chapter_names = {{ localizedChapterTitles|tojson }}
+    var featured_chapters = {{ featured_quotes|tojson }}
+
+    var randomChapter="";
+    
+    // Only run this if the page is reloaded for performance reasons.
+    // Support both newer PerformanceNavigationTiming API for newer browsers
+    // and older Navigation Timing API for older browsers
+    if ((performance.getEntriesByType("navigation")[0] && performance.getEntriesByType("navigation")[0].type === "reload")
+        ||
+         (performance.navigation && performance.navigation.type === performance.navigation.TYPE_RELOAD)) {
+      const keys = Object.keys(featured_chapters)
+      const randIndex = Math.floor(Math.random() * keys.length)
+      randomChapter = keys[randIndex];
+    }
+  </script>
+  <section id="featured-chapter" class="featured-chapter">
+    <div class="featured-chapter-content">
+      <h2 class="title title-center">{{ self.featured_chapter() }}</h2>
+      <h3 class="featured-chapter-name">{{ localizedChapterTitles[random_chapter] }}</h3>
+      <blockquote id="featured-quote">
+        {{ featured_chapter_quotes.get('quote')|safe }}
+      </blockquote>
+      <div class="featured-chapter-content-data">
+        <div class="featured-chapter-content-data-item">
+          <div id="featured-stat-1" {{ stat1_hidden }}>{{ featured_chapter_quotes.get('stat_1')|safe }}</div>
+          <div id="featured-stat-label-1" {{ stat1_hidden }}>{{ featured_chapter_quotes.get('stat_label_1')|safe }}</div>
+        </div>
+        <div class="featured-chapter-content-data-item">
+          <div id="featured-stat-2" {{ stat2_hidden }}>{{ featured_chapter_quotes.get('stat_2')|safe }}</div>
+          <div id="featured-stat-label-2" {{ stat2_hidden }}>{{ featured_chapter_quotes.get('stat_label_2')|safe }}</div>
+        </div>
+        <div class="featured-chapter-content-data-item">
+          <div id="featured-stat-3" {{ stat3_hidden }}>{{ featured_chapter_quotes.get('stat_3')|safe }}</div>
+          <div id="featured-stat-label-3" {{ stat3_hidden }}>{{ featured_chapter_quotes.get('stat_label_3')|safe }}</div>
+        </div>
+      </div>
+      <a id="featured-link" href="{{ url_for('chapter', year=year, chapter=random_chapter, lang=lang) }}" class="btn">
+        {{ readChapter(featured_chapter_name) }}
+      </a>
+    </div>
+  </section>
+  {% endif %}
+  <script nonce="{{ csp_nonce() }}">
+
+    // Replace the quote with the randomly generated one
+    if (randomChapter) {
+      var featured_quote = document.querySelector('#featured-quote');
+      if (featured_quote) {
+        featured_quote.innerHTML = featured_chapters[randomChapter].quote;
+      }
+
+      var featured_titles = document.querySelectorAll('.featured-chapter-name');
+      for (var i = 0; i < featured_titles.length; i++) {
+        featured_titles[i].innerHTML = chapter_names[randomChapter];
+      }
+
+      for (let i=1; i <= 3; i++) {
```

Comment:
Any specific reason we are using `let` only in this one instance whereas we use `var` almost everywhere else?

Reply:
Good spot! Should be `var` to support older browsers. About to write a doc on this for all our new developers!!

Reply:
Fixed. Though IE allows `let` in a for loop, which is why I didn't spot this earlier when testing in there! Still better to be consistent.